unlock-app/src/__tests__/reducers/metadataReducer.test.ts

```diff
@@ -53,20 +69,11 @@ describe('metadataReducer', () => {
 
   it('adds a lock to the state with metadata', () => {
     expect.assertions(1)
-    const result = reducer(undefined, {
-      type: GOT_METADATA,
-      lockAddress: 'my address',
-      keyId: '7',
-      data: {
-        public: {
-          color: 'blue',
-        },
-      },
-    })
+    const result = reducer(undefined, gotBulkMetadata(lockAddress, metadata))
 
     expect(result).toEqual({
-      'my address': {
-        '7': {
+      [lockAddress.toLowerCase()]: {
```

Comment:
haha! I see how this is biting you too ;p

Reply:
Took me longer than I care to admit to realize why my filter was coming back empty